src/api-client.ts

```diff
@@ -57,9 +57,25 @@ export class RevAiApiClient {
             { 'Content-Type': 'application/json' }, 'json', options);
     }
 
-    async submitJobLocalFile(filename: string, options?: RevAiJobOptions): Promise<RevAiApiJob> {
+    async submitJobFileStream(
+        fileStream: Buffer | Readable,
```

Comment:
`Buffer` isn't exactly a stream but it can be treated in the same way. Not sure if I should change the method name somehow

Reply:
perhaps `submitJobWithAudioData(audioData: Buffer | Readable)`?

using `FileStream` for method name / property name is somewhat misleading because it doesn't have to come rom a file (presumably the point of the method is to allow you to not have to get the data from a file)